entrypoint

```diff
@@ -201,7 +203,20 @@ else
 
     # wait, unless RUN_ONCE is set
     if [ -z "${RUN_ONCE}" ]; then
-      sleep $(($DB_DUMP_FREQ*60))
+      current_time=$(date +"%s")
```

Comment:
I am confused by the algorithm here. Why not just do the following:

1. Beginning of each backup, calculate (pseudo-code) `next_run_time=$current_time + $DB_DUMP_FREQ*60`, e.g. if we start at 14:00 and want 24 hours, then `next_run_time` = "14:00 tomorrow"
2. End of each backup calculate `current_time`, e.g. if backup took 2 hours, then `current_time` = "16:00"
3. Calculate `wait_time = $next_run_time - $current_time`
4. `sleep $wait_time`

?

Reply:
Ok, the .ignore can be skipped indeed because it doesn't need to be updated. I will remove it from the pull request.
The algorithm I implemented will also take care of the situation where a user defined to create a backup every 15 minutes and where the backup itself will take 40 minutes after a while. 
In this case, it will start in 5 minutes (3 times 15 minutes) which is more clear to the user when he looks at the timestamps of the created backups.
In the algorithm that you describe, wait_time can become negative which is not allowed to pass to the sleep command. You can implement a check around it and use 0 as a value but in this case, the database will continually be in backup mode.

Reply:
> The algorithm I implemented will also take care of the situation where a user defined to create a backup every 15 minutes and where the backup itself will take 40 minutes after a while

Reply:
Ok, thanks